### DISCUSSION TITLE
Wie wird GitHub zum Lernen verwendet ?

### DISCUSSION THREAD

**Discussion by uwereinerprochnau** [community/community#65282] [GitHub Education] [answered]
### Hi everyone,

Hallo, ich bin ganz neu hier. Komme noch überhaupt nicht zurecht. Kann mir bitte Jemand helfen, den richtigen Einstieg zu finden ?

**Comment by arafajs** [comment, marked as answer]
GitHub can be used for learning in many ways, including:

- **Learning the basics of version control.** GitHub is a version control system, which means it allows you to track changes to your code over time. This can be a valuable skill for any programmer, as it helps you to keep your code organized and to revert to previous versions if necessary.
- **Learning from other developers.** GitHub is home to millions of open source projects, which means you can find code for just about anything you're interested in learning. You can also read through the comments and discussions on these projects to learn from other developers' experiences.
- **Collaborating on projects with other developers.** GitHub makes it easy to collaborate on projects with other developers, even if you're not in the same place. This is a great way to learn from others and to get feedback on your own code.
- **Building your portfolio.** GitHub can be used to showcase your coding skills to potential employers or clients. By creating repositories for your projects and keeping them up-to-date, you can show off your skills and experience.
- **Learning about new technologies.** GitHub is a great way to learn about new technologies. Many projects on GitHub use cutting-edge technologies, so you can stay up-to-date on the latest trends.

**Reply by uwereinerprochnau** [reply]
Das ist eine sehr ausführliche und professionelle Antwort auf meine Frage. Ich bin begeistert. Zudem bedanke ich mich bei den Entwickler für Ihre aufwendigen Bemühungen.
